.travis-ci.sh

```diff
@@ -11,6 +11,18 @@ COVERITY_SCAN_BUILD_URL="https://scan.coverity.com/scripts/travisci_build_coveri
 
 PYCHECKER_BLACKLIST="threading,unittest,cmd,optparse,google,google.protobuf,ssl,fftpack,lapack_lite,mtrand"
 
+LINT_BLACKLIST=$(cat <<EOLBL
```

Comment:
Could we make the EOLBL and the one below more consistent, I guess being different may be safest, but consistent names would be nice.

Having just double-checked, the - at the start of the spelling one suppresses the leading tabs, although I'm not sure if we really need it:
http://tldp.org/LDP/abs/html/here-docs.html